.github/workflows/ci.yaml

```diff
@@ -7,8 +7,21 @@ on:
 jobs:
   test:
```

Comment:
Unrelated but should we rename this since it's not just tests?

Reply:
I still feel it's mainly testing related. Linting (prettier) and checking if the project builds is in my opinion part of testing. But feel free to suggest a different name though.

Reply:
I mean we can keep it as is. From the top of my head I don't have a better name. Personally I would have different jobs and separate some of the stuff but like I said, we can keep it as is.